central/convert/internaltov2storage/compliance_rule.go

```diff
@@ -0,0 +1,53 @@
+package internaltov2storage
+
+import (
+	"strings"
+
+	"github.com/stackrox/rox/generated/internalapi/central"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+const (
+	standardsKey = "policies.open-cluster-management.io/standards"
+
+	controlAnnotationBase = "control.compliance.openshift.io/"
+)
+
+// ComplianceOperatorRule converts message from sensor to V2 storage
+func ComplianceOperatorRule(sensorData *central.ComplianceOperatorRuleV2, clusterID string) *storage.ComplianceOperatorRuleV2 {
+	fixes := make([]*storage.ComplianceOperatorRuleV2_Fix, 0, len(sensorData.Fixes))
+	for _, fix := range sensorData.Fixes {
+		fixes = append(fixes, &storage.ComplianceOperatorRuleV2_Fix{
+			Platform:   fix.GetPlatform(),
+			Disruption: fix.GetDisruption(),
+		})
+	}
+
+	// TODO:  figure out where to grab this.  Basically is policies.open-cluster-management.io/standards legit to
```

Comment:
@rhmdnd Does this make sense and does it make sense to use that key?

Reply:
I believe so since that appears to be where the "benchmark" or "standard" is referenced from within the rule:

```
 $ oc get rules -n openshift-compliance ocp4-classification-banner   -o json | jq .metadata.annotations
{
  "compliance.openshift.io/image-digest": "pb-ocp45kdr4",
  "compliance.openshift.io/profiles": "ocp4-stig-v1r1,ocp4-stig",
  "compliance.openshift.io/rule": "classification-banner",
  "control.compliance.openshift.io/NIST-800-53": "AC-8",
  "policies.open-cluster-management.io/controls": "AC-8",
  "policies.open-cluster-management.io/standards": "NIST-800-53"
}
```